R/string_operation.R

```diff
@@ -373,3 +373,18 @@ get_sentiment <- function(text){
   loadNamespace("sentimentr")
   sentimentr::sentiment_by(text)$ave_sentiment
 }
+
+#' Wrapper function for readr::parse_parse_character.
+#' @param text to parse
+#' @export
+parse_character <- function(text, ...){
+  loadNamespace("readr")
+  tryCatch(
```

Comment:
Rather than catching all errors, which can be possible caused by other reasons, first check if input is not character by is.character() and if so, fall back to as.character.